backend/app/api/admin_routes/upload.py

```diff
@@ -33,6 +34,14 @@ def upload_files(
                 status_code=status.HTTP_400_BAD_REQUEST,
                 detail="File name cannot be empty",
             )
+        if SiteSetting.setting_exists("upload_max_body_size"):
+            sys_upload_max_body_size = SiteSetting.get_setting("upload_max_body_size")
+            if file.size > sys_upload_max_body_size:
+                raise HTTPException(
+                    status_code=status.HTTP_413_REQUEST_ENTITY_TOO_LARGE,
+                    detail=f"File size({file.size}) exceeds maximum allowed size({sys_upload_max_body_size})",
```

Comment:
Can me make the file size humam-readble?

Reply:
ok